pkg/kube/secrets.go

```diff
@@ -142,7 +143,7 @@ type secretsReader struct {
 }
 
 func (r *secretsReader) ListByLocalObjectReferences(ctx context.Context, refs []corev1.LocalObjectReference, ns string) ([]corev1.Secret, error) {
-	secrets := make([]corev1.Secret, 0)
+	var secrets []corev1.Secret
```

Comment:
imho it's not equal too, and it's important for tests

Reply:
updated.